File: app/src/main/java/com/example/balbripcu/ViewCourseUnits.java
package com.example.balbripcu;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class ViewCourseUnits extends AppCompatActivity {

      private Button backBtn ;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_print_course_units);

        backBtn = findViewById(R.id.back_btn);

        backBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(getApplicationContext(), MainActivity.class));
            }
        });
    }
}

File: app/src/main/java/com/example/balbripcukotlin/ViewCourseUnits.kt
package com.example.balbripcukotlin

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class ViewCourseUnits : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {


        lateinit var backButton: Button

        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_view_course_units)

        backButton = findViewById<Button>(R.id.back_btn)
        backButton.setOnClickListener{
            //Toast.makeText(this,"View Button clicked!",Toast.LENGTH_SHORT.show())
            val intent = Intent(this,MainActivity::class.java)
            startActivity(intent)
        }
    }
}